src/rebar_prv_dialyzer.erl

```diff
@@ -23,6 +23,7 @@
 -spec init(rebar_state:t()) -> {ok, rebar_state:t()}.
 init(State) ->
     Opts = [{update_plt, $u, "update-plt", boolean, "Enable updating the PLT. Default: true"},
+            {update_base_plt, $b, "update-base-plt", boolean, "Enable updating the base PLT. Default: true"},
```

Comment:
> But really we only update the base plt if the project plt does not exist.
Exactly the situation that I'm trying to avoid. What about `base_plt_read_only` ? Any suggestions on the naming?